### DISCUSSION TITLE
Ansible with pyenv

### DISCUSSION THREAD

**Discussion by davosian** [pyenv/pyenv#1823] [Q&A]
Based on Python 3.9.1, I installed ansible successfully with `pip install ansible==2.10.4`, but I am unable to find the binary afterwards meaning I cannot run it.

`pip show ansible` lists `~/.local/lib/python3.9/site-packages` as installation path but at this location I only find different ansible folders and no binaries, nor a `bin/` directory.

Anyone knows the path where `pyenv` installs the pip binaries?

**Comment by anton-petrov** [comment]
`pip show ansible`  on Ubuntu 20.04

> pip show ansible
Name: ansible
Version: 3.0.0
Summary: Radically simple IT automation
Home-page: https://ansible.com/
Author: Ansible, Inc.
Author-email: info@ansible.com
License: GPLv3+
**Location: /home/anton/.pyenv/versions/3.9.2/lib/python3.9/site-packages
![Screenshot 2021-02-22 100049](https://user-images.githubusercontent.com/2428315/108673524-dd629600-74f4-11eb-85a8-7182ed3840f1.png)
**
Requires: ansible-base
Required-by:

Please, provide your system configuration

**Reply by davosian** [reply]
Thanks for pointing this out @anton-petrov. I do see the same setup, however this is only a directory and not the binary.
The background of my request is that I need to set the `PATH` variable so that the `ansible` binary is found. At least in my `zsh` environment on macOS 11.2.1, it otherwise does not work i.e. it does not find ansible, ansible-playbook etc.
